Formula/pagmo.rb

```diff
@@ -1,8 +1,8 @@
 class Pagmo < Formula
   desc "Scientific library for massively parallel optimization"
   homepage "https://esa.github.io/pagmo2/"
-  url "https://github.com/esa/pagmo2/archive/v2.15.0.tar.gz"
-  sha256 "07977be690b512ea11fc40d5b8bfa0a7a8507ce9053a615c2bc4725d355ef9a8"
+  url "https://github.com/esa/pagmo2/archive/v2.16.0.tar.gz"
+  sha256 "076918ca975e2b45eedd85b65da0de650d4bd0b3f86182c0c144c7fdc191185b"
   license "GPL-3.0"
```

Comment:
```suggestion
  license "GPL-3.0-or-later"
```
(guessed from https://github.com/esa/pagmo2/blob/cc82c955f6027fba8a403550fc83c05d2f857afb/src/algorithm.cpp#L10)